### PR TITLE
MEN-3926: document the mTLS device authentication flow

### PR DESCRIPTION
**Pull request by tranchitella**
Changelog: title

Signed-off-by: Fabio Tranchitella <fabio.tranchitella@northern.tech>

### CODE REVIEW

**Review by kacf — DISMISSED** [review]
Nice format, "mmd", I didn't know about it. I couldn't find much info on it, how is it converted to PNG?

**Comment by tranchitella** [comment]
@kacf mermaid, it's a javascript package but you can use the online editor here:
https://mermaid-js.github.io/mermaid-live-editor
Shall we document it somewhere?

**Comment by kacf** [comment]
Wait, Github tricked me, I only looked at half the PR, don't merge yet.

File: 02.Overview/13.Device-authentication/docs.md [hunk 1]
```diff
@@ -51,10 +51,12 @@ given time.
 
 ## Authorization Flows
 
-Mender provides two possible authorization flows. Both involve a user's explicit
+Mender provides three possible authorization flows. Two of them involve a user's explicit
 consent to authorize a Device via the UI or Device Authentication API, but they differ
-in the order of events and intended use cases. Below is a detailed breakdown of
-each.
+in the order of events and intended use cases. The third one, available as Enterprise
```
Comment (kacf) [inline 02.Overview/13.Device-authentication/docs.md:56]: "..as **an** Enterprise feature.."

**Review by kacf — APPROVED** [review]
Just a minor grammatical typo, I trust you'll fix it, so approving!

**Comment by kacf** [comment]
> @kacf mermaid, it's a javascript package but you can use the online editor here:
> https://mermaid-js.github.io/mermaid-live-editor
> Shall we document it somewhere?

Maybe just in the commit message is enough?

File: 02.Overview/13.Device-authentication/docs.md (hunk 1 above)
Comment (tranchitella) [reply 02.Overview/13.Device-authentication/docs.md:56]: fixed

**Comment by tranchitella** [comment]
> > @kacf mermaid, it's a javascript package but you can use the online editor here:
> > https://mermaid-js.github.io/mermaid-live-editor
> > Shall we document it somewhere?
> 
> Maybe just in the commit message is enough?

Right, added a note in the commit message.

**Pull request merged**